packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -2586,3 +2586,45 @@ class RadioButton<T> extends StatelessWidget {
     );
   }
 }
+
+class ContextMenuButton extends StatelessWidget {
+  const ContextMenuButton({
+    this.style,
+    this.gaScreen,
+    this.gaItem,
+    required this.menu,
+  });
+
+  static const double width = 14;
+
+  final TextStyle? style;
+  final String? gaScreen;
+  final String? gaItem;
+  final List<Widget> menu;
+
+  @override
+  Widget build(BuildContext context) {
+    return MenuAnchor(
+      menuChildren: menu,
+      builder:
+          (BuildContext context, MenuController controller, Widget? child) {
+        return SizedBox(
+          width: ContextMenuButton.width,
```

Comment:
Nit: can this not just be `width` since we're already in the scope of `ContextMenuButton`?